### DISCUSSION TITLE
Import from Notion not available for files with size more than 26.21 MB

### DISCUSSION THREAD

**Discussion by dmpanch** [outline/outline#3642] [Self Hosting] [answered]
**To Reproduce**
1. Go to 'Settings -> Import'
2. Click on 'Import' from Notion
3. Choose zip file exported from Notion with size more than 26.21 MB
4. Get error

```
{
    "ok": false,
    "error": "validation_error",
    "status": 400,
    "message": "Sorry, this file is too large – the maximum size is 26.21 MB"
}
````

**Expected behavior**
The file is imported correctly.
Also it would be great to move a such setting to an environment variable or limit the maximum file size by NGINX configuration.

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.

**Outline (please complete the following information):**
- Install: selfhosted
- Version: 8a29a35

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.6 LTS
 - Browser: Safari
 - Version: 15.5 (17613.2.7.1.8)

**Comment by tommoor** [comment, marked as answer]
You're looking for `AWS_S3_UPLOAD_MAX_SIZE` env variable

**Reply by dmpanch** [reply]
Thanks, I thought about the Minio settings, but did not connect with the fact that the same variable is used.
It was also not obvious that the same variable could be used for backups, not only for static files for documentation.
